juno/assetlist.json

```diff
@@ -1307,6 +1307,50 @@
       "logo_URIs": {
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/void.png"
       }      
+    },
+    {
+      "description": "Slica",
+      "type_asset": "cw20",
+      "address": "juno10vgf2u03ufcf25tspgn05l7j3tfg0j63ljgpffy98t697m5r5hmqaw95ux",
+      "denom_units": [
+        {
+          "denom": "cw20:juno10vgf2u03ufcf25tspgn05l7j3tfg0j63ljgpffy98t697m5r5hmqaw95ux",
+          "exponent": 0
+        },
+        {
+          "denom": "slica",
+          "exponent": 6
+        }
+      ],
+      "base": "cw20:juno10vgf2u03ufcf25tspgn05l7j3tfg0j63ljgpffy98t697m5r5hmqaw95ux",
+      "name": "Slica",
```

Comment:
Looks like this token is called 'Silica', not 'slica'. Can you please correct this? https://www.mintscan.io/juno/wasm/contract/juno10vgf2u03ufcf25tspgn05l7j3tfg0j63ljgpffy98t697m5r5hmqaw95ux